sale_commission/migrations/13.0.1.0.0/post-migration.py

```diff
@@ -42,3 +51,24 @@ def migrate(env, version):
     """
         ).format(sql.Identifier(openupgrade.get_legacy_name("invoice_line_id"))),
     )
+    openupgrade.logged_query(
+        env.cr,
+        """
+            UPDATE account_move am
+            SET commission_total = ai.commission_total
+            FROM account_invoice ai
+            WHERE ai.id = am.old_invoice_id
+                AND ai.commission_total IS NOT NULL
+                AND ai.commission_total != 0.0
+        """,
+    )
+    openupgrade.logged_query(
+        env.cr,
+        """
+            UPDATE account_move_line aml
+            SET commission_free = pt.commission_free
+            FROM product_product pp
+                INNER JOIN product_template pt ON pt.id = pp.product_tmpl_id
+            WHERE aml.product_id = pp.id AND pt.commission_free = TRUE
```

Comment:
Just FYI, this is the same:

```suggestion
            WHERE aml.product_id = pp.id AND pt.commission_free
```

Reply:
I don't think you should do this, but take it from the source account.invoice.line, as the check may vary over time, having different values in the resulting documents.

Reply:
Sure, thanks!!! I didn't notice the information could be found in invoice line. It has more sense to import data from there in order to keep the same information.